alpha/apps/kaltura/lib/multiLingualUtils.php

```diff
@@ -260,4 +260,26 @@ public static function isMultiLingualRequest ($newMapping)
 		}
 		return isset(kCurrentContext::$language);
 	}
+	
+	public static function getFieldValue($dbObject, $fieldName)
+	{
+		$value = $dbObject->getDefaultFieldValue($fieldName);
+		$mapping = self::getMultiLanguageMapping($dbObject);
+		if (!$mapping || ($mapping == ''))
+		{
+			return $value;
+		}
+		return self::concatMultiLingualValuesForField($value, $mapping, $fieldName);
+	}
+	
+	protected static function concatMultiLingualValuesForField($value, $mapping, $fieldName)
+	{
+		$mapping = json_decode($mapping, true);
+		$result = $value;
+		foreach ($mapping[$fieldName] as $languageKey => $currentFieldValue)
+		{
+			$result = $result. ',' .$currentFieldValue;
```

Comment:
just add space, not a ',' char

Reply:
I think that if we separate using spaces there will be a problem with names and descriptions with spaces. Also, tags are separated by ','.